src/Chunk.cpp

```diff
@@ -1864,13 +1863,41 @@ void cChunk::RemoveClient(cClientHandle * a_Client)
 		}
 	}
 
+	// All clients were removed. No one sees this chunk any more.
+	// Delete all hostiles inside it.
+	if (!HasAnyClients())
+	{
+		for (cEntityList::iterator itr = m_Entities.begin(); itr != m_Entities.end(); ++itr)
```

Comment:
The C++ standards committee expresses its dissatisfaction at the non-use of C++11 features here.